push-notifications/src/database.js

```diff
@@ -1,5 +1,5 @@
 import mysql from 'mysql';
-import {db} from '../conf/config';
+import { db } from './config/index';
```

Comment:
you don't need to specify `index` in the path when the file itself is `index.js`